Logging/Logging/BrainstormSessions/Controllers/HomeController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using BrainstormSessions.Core.Interfaces;
+using BrainstormSessions.Core.Model;
+using BrainstormSessions.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace BrainstormSessions.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly IBrainstormSessionRepository _sessionRepository;
+        private readonly ILogger<HomeController> _logger;
```

Comment:
It's a good practice as well to log all incoming request via middleware of action filter (I would advise the second option since it can include some configurable items via settings params and substitute business logic parts).
You can include the following log parts when request it processed:
* URL (protocol + domain + route + query params), 
* HTTP Request Method
* HTTP Request Headers
* Body (if it exists)

Response items that should be logged:
* HTTP Status code numeric value
* HTTP Status code message
* HTTP Response headers
* Response in JSON format

Of course, logging of body/response is some cases is not preferable since it may contain user sensitive information, in this scenario your can provide "blacklist" to exclude some endpoints parts from logging